File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/EventDao.java
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api;

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface;
import ru.kami.gis.afisha.schedule.core.domain.EventEntity;

import java.util.Optional;

/**
 * @author Daniil.Makarov
 */
public interface EventDao extends JdbcDaoInterface {
    Optional<EventEntity> findEventById(Long id);
}

File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/EventDao.kt
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface
import ru.kami.gis.afisha.schedule.core.domain.EventEntity

/**
 * @author Daniil.Makarov
 */
interface EventDao : JdbcDaoInterface {
    fun findEventById(id: Long): EventEntity?
}